src/coreclr/tools/aot/crossgen2/crossgen2.props

```diff
@@ -14,7 +14,7 @@
     <Configurations>Debug;Release;Checked</Configurations>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <!-- Trim all dependent assemblies as though they were all marked IsTrimmable. -->
-    <TrimmerDefaultAction>link</TrimmerDefaultAction>
+    <TrimMode>full</TrimMode>
```

Comment:
Do we need this line if this is now the default?

Reply:
No, we don't, I was just being explicit.